by-laws.md

```diff
@@ -1,1065 +1,295 @@
 # BY-LAWS OF NODE.JS FOUNDATION
 
-## ARTICLE I - NAME, PURPOSE AND OFFICES
+## ARTICLE I NAME, PURPOSE AND OFFICES
+
+### Section 	1.1  	Name  
+The name of the corporation is “Node.js Foundation”, and the corporation is referred to in these By-laws as the “Foundation”.  
```

Comment:
@mrhinkle ... from a formatting point of view, it would be significantly easier to review if there were whitespace between the headers and paragraphs... e.g.

```
### Section 1.1 Name

The name of the ...

```